Tema4/src/Animal_Product.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class Animal_Product extends Product implements Validity {
+    private LocalDate expDate;
+    private double freezeTemp;
+
+    public Animal_Product (LocalDate expDate, String id, double price, int quantity, double weight){
+        super(id, price, quantity, weight);
+        this.expDate = expDate;
+    }
+
+    public Animal_Product (LocalDate expDate, String id, double price, int quantity, double weight, double freezeTemp){
+        super(id, price, quantity, weight);
+        this.expDate = expDate;
+        this.freezeTemp = freezeTemp;
+    }
+
+    public double getFreezeTemp(){
+        return freezeTemp;
+    }
+
+    public void setFreezeTemp(double freezeTemp){
+        this.freezeTemp = freezeTemp;
+    }
+
+    public LocalDate getExpDate(){
+        return expDate;
+    }
+
+    public void setExpDate(LocalDate expDate){
+        this.expDate = expDate;
+    }
+
+    public static void main(String[] args) {
```

Comment:
this looks like forgotten test code... please try to pay  more attention before commiting